e2etests/test/module_messaging.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var ModuleIdentityHelper = require('./module_identity_helper.js');
+var EventHubReceiverHelper = require('./eventhub_receiver_helper');
+var Message = require('azure-iot-common').Message;
+var assert = require('chai').assert;
+var debug = require('debug')('e2etests:module-messaging');
+var Amqp = require('azure-iot-device-amqp').Amqp;
+var AmqpWs = require('azure-iot-device-amqp').AmqpWs;
+//var Mqtt = require('azure-iot-device-mqtt').Mqtt;
+//var MqttWs = require('azure-iot-device-mqtt').MqttWs;
+
+var transportsToTest = [ Amqp, AmqpWs ];
+
+describe('module messaging', function() {
+  this.timeout(46000);
+
+  transportsToTest.forEach(function(Transport) {
+    describe('using ' + Transport.name, function() {
+      var testModule = {};
+
+      before(function(done) {
+        ModuleIdentityHelper.createModule(testModule, Transport, done);
+      });
+
+      after(function(done) {
+        ModuleIdentityHelper.cleanUpAfterTest(testModule, done);
+      });
+
+      it ('Can send from service to module input', function(done) {
```

Comment:
>  [](start = 8, length = 1)

nit: unnecessary space (isn't jshint angry about that?)

Reply:
I saw someone else doing this in our code and copied it.  Wasn't it you?  

---
In reply to: [180596656](https://github.com/Azure/azure-iot-sdk-node/pull/271#discussion_r180596656) [](ancestors = 180596656)

Reply:
might be, inadvertently. it's not important.

---
In reply to: [181216828](https://github.com/Azure/azure-iot-sdk-node/pull/271#discussion_r181216828) [](ancestors = 181216828,180596656)